### ISSUE TITLE
How to generateOpenApiDocument?

### ISSUE THREAD

**Issue by Blankeos** [jlalmes/trpc-openapi#276]
This is a noob question but how do I use this particular code snippet for creating an openapi document? Or what route do I need to access to access documentation? I'm using this trpc-openapi with Next by the way.

```ts
import { generateOpenApiDocument } from 'trpc-openapi';

import { appRouter } from '../appRouter';

/* 👇 */
export const openApiDocument = generateOpenApiDocument(appRouter, {
  title: 'tRPC OpenAPI',
  version: '1.0.0',
  baseUrl: 'http://localhost:3000',
});
```

**Comment by toddjudd** [comment]
That code should be placed in an openapi.json.ts file in your api directory. Once there and once your endpoints are set up you can navigate to that page same as any other endpoint to see the json. 

example: `http:\\localhost;3000\api\openapi.json`

**Comment by Blankeos** [comment]
Thanks @toddjudd!

I also noticed that you can paste the the contents of **openapi.json.ts** into [editor.swagger.io](https://editor.swagger.io/) and see well-organized UI for the documentation. I'm thinking of display that same Swagger UI within my app via a NextJS Route. Is this possible? Or is this not recommended to do?

**Comment by toddjudd** [comment]
I've never done it myself, but it sounds like a solid choice. Might be some helpful information here. https://next-swagger-doc.productsway.com/

**Comment by Blankeos** [comment]
Thanks for this, I was able to successfully do it.

I opted for a local dev environment approach for viewing the api documentation, so that I won't need to create an API specifically for viewing documentation.

I got the idea from [next-swagger-doc](https://next-swagger-doc.productsway.com/), but I used [swagger-ui-cli](https://www.npmjs.com/package/swagger-ui-cli) instead.

Basically, I write the openApiDocument as a JSON and view it with the swagger-ui-cli.
